.github/actions/bazel-test-all/action.yaml

```diff
@@ -102,39 +102,12 @@ runs:
             bazel ${{ inputs.BAZEL_COMMAND }} "${bazel_args[@]}"
 
       - name: Upload to S3
-        uses: ./.github/actions/bazel
+        uses: ./.github/actions/upload-artifacts
         if: inputs.upload-artifacts == 'true' && inputs.CLOUD_CREDENTIALS_CONTENT != ''
         with:
-          run: |
-            stamp_build='${{ inputs.stamp-build }}'
-
-            if [[ $stamp_build != "true" ]]; then
-              echo "refusing to upload non-stamped artifacts"
-              exit 1
-            fi
```

Comment:
I see this stamp check was not migrated over. Is that intentional?

Reply:
It is indeed, but nice catch! The action has no knowledge of the config so I'll let the caller deal with that.